packages/core/src/utils/TypeUtil.ts

```diff
@@ -169,12 +175,12 @@ export interface TransactionContent {
 
 export interface TransactionPreview {
   url: string
-  content_type?: any
+  content_type?: null
```

Comment:
I think it can be a video object as well or gif, so possible we should check for the MIME type (eg. `image/png`)